src/WebAuthn/Types.hs

```diff
@@ -41,12 +41,16 @@ module WebAuthn.Types (
   , AuthenticatorSelection (..)
   , UserVerification (..)
   , PubKeyCredAlg (..)
+  , AuthnSel
+  , BiometricPerfBounds
```

Comment:
They should have been exported indeed, but without constructors they don't seem very useful.